src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -144,4 +132,26 @@ public JLabel getHeaderLabel() {
             return new JLabel(getDisplayName());
         }
     }
+
+    public boolean isResolved(BibEntry entry) {
```

Comment:
Yes, that is correct. As the class is called MainRefColumn, however, it is
the first of the column fields (`journal/booktitle`).